src/functionalTest/resources/requests/bulk-scan-a59-transform-input.json

```diff
@@ -0,0 +1,46 @@
+{
+  "ocr_data_fields":[
+    {
+      "name":"applicant_firstName",
+      "value":"applicant1_firstName"
+    },
+    {
+      "name":"applicant_lastName",
+      "value":"applicant1_lastName"
```

Comment:
Why is is referring to applicant1_lastName in some places and directly to value for example for telephone number

Reply:
we can put any value, value could be anything